cypress/support/pages/contactForm.pageObject.js

```diff
@@ -3,36 +3,49 @@ import PageObject from '../PageObject';
 class ContactFormPageObject extends PageObject {
   url = '/index.html';
 
-  get emailField() {
-    return cy.get('#recipient-email');
+  get placeOrderBtn() {
+    return cy.get(".btn", "Place Order");
+  }
+  clickPlaceOrderBtn() {
+    this.placeOrderBtn.click();
   }
-
   get nameField() {
-    return cy.get('#recipient-name');
+    return cy.get("#name").should('exist')
   }
-
-  get messageField() {
-    return cy.get('#message-text');
+  get country_field() {
+    return cy.get("#country");
   }
-
-  get sendMessageBtn() {
-    return cy.contains('.btn', 'Send message');
+  get city_field() {
+    return cy.get("#city");
   }
-
-  typeEmail(email) {
-    this.emailField.type(email, { force: true });
+  
+  get card_field() {
+    return cy.get("#card");
   }
-
-  typeName(name) {
-    this.nameField.type(name, { force: true });
+  get month_field() {
+    return cy.get("#month");
   }
-
-  typeMessage(message) {
-    this.messageField.type(message);
+  get year_field() {
+    return cy.get("#year");
+  }
+  get purchase_btn() {
+    return cy.get(".btn", "Purchase");
+  }
+  click_purchase_btn() {
+    this.purchase_btn.click();
   }
 
-  clickOnSendMessageBtn() {
-    this.sendMessageBtn.click();
+  get successful_message_modal() {
+    return cy.get(".sweet-alert");
+  }
+  assert_succesful_message() {
```

Comment:
Function names should be verbs or verb phrases. Therefore, 'assert_succesful_message' should be 'assertSuccessfulMessage'.